common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -640,3 +640,39 @@ def definition_to_xml(self, resource_fs):
             if field.is_set_on(self):
                 xml_object.set(field_name, unicode(field.read_from(self)))
         return xml_object
+
+
+class LibrarySummary(object):
+    """
+    A library summary object which contains the fields required for library listing on studio.
+    """
+
+    def __init__(self, library_locator, display_name):
+        """
+        Initialize LibrarySummary
+
+        Arguments:
+        library_locator (LibraryLocator):  LibraryLocator object of the library.
+
+        display_name (unicode): display name of the library.
+        """
+        self.display_name = display_name if display_name else u"Empty"
```

Comment:
would `Empty` need to be translated?

Reply:
We generally do not translate display names of libraries/courses. Should we consider this as a special case and translate it? What are your thoughts on this?

Reply:
@adampalay is that makes sense to not translate `display_name`? what are your thoughts?

Reply:
`"Empty"` isn't a display name though. It's a placeholder that indicates that no display name is provided.